frontend/sass/_modal.scss

```diff
@@ -22,21 +22,52 @@
   flex-direction: column;
 }
 
+// This styling applies to desktop Internet Explorer
+.scrollable {
+  overflow-y: auto;
+  max-width: 640px;
+}
+
 .jf-modal-dialog {
   cursor: default;
+  overflow-y: initial !important;
```

Comment:
Is there any way to make this rule without the `!important`?  It's generally preferable to avoid it, though there [are some exceptions](https://css-tricks.com/when-using-important-is-the-right-choice/).